core/blockly.js

```diff
@@ -430,11 +492,85 @@ exports.Block = Block;
 exports.BlockDragger = BlockDragger;
 exports.BlockDragSurfaceSvg = BlockDragSurfaceSvg;
 exports.BlockSvg = BlockSvg;
+exports.Bubble = Bubble;
+exports.BubbleDragger = BubbleDragger;
 exports.CollapsibleToolboxCategory = CollapsibleToolboxCategory;
+exports.Comment = Comment;
+exports.ComponentManager = ComponentManager;
+exports.Connection = Connection;
+exports.ConnectionChecker = ConnectionChecker;
+exports.ConnectionDB = ConnectionDB;
+exports.ContextMenu = ContextMenu;
+exports.ContextMenuItems = ContextMenuItems;
+exports.ContextMenuRegistry = ContextMenuRegistry;
+exports.Css = Css;
 exports.Cursor = Cursor;
+exports.DeleteArea = DeleteArea;
+exports.DragTarget = DragTarget;
+exports.DropDownDiv = DropDownDiv;
+exports.Extensions = Extensions;
+exports.FieldAngle = FieldAngle;
+exports.Flyout = Flyout;
+exports.FlyoutButton = FlyoutButton;
+exports.Generator = Generator;
+exports.Gesture = Gesture;
+exports.Grid = Grid;
+exports.HorizontalFlyout = HorizontalFlyout;
+exports.Icon = Icon;
+exports.Input = Input;
+exports.InsertionMarkerManager = InsertionMarkerManager;
 exports.Marker = Marker;
+exports.MarkerManager = MarkerManager;
+exports.Menu = Menu;
+exports.MenuItem = MenuItem;
+exports.MetricsManager = MetricsManager;
+exports.Mutator = Mutator;
+exports.Names = Names;
+exports.Options = Options;
+exports.Procedures = Procedures;
+exports.RenderedConnection = RenderedConnection;
+exports.Scrollbar = Scrollbar;
+exports.ScrollbarPair = ScrollbarPair;
+exports.ShortcutItems = ShortcutItems;
+exports.ShortcutRegistry = ShortcutRegistry;
 exports.TabNavigateCursor = TabNavigateCursor;
+exports.Theme = Theme;
+exports.ThemeManager = ThemeManager;
 exports.Toolbox = Toolbox;
 exports.ToolboxCategory = ToolboxCategory;
 exports.ToolboxItem = ToolboxItem;
 exports.ToolboxSeparator = ToolboxSeparator;
+exports.Tooltip = Tooltip;
+exports.Touch = Touch;
+exports.TouchGesture = TouchGesture;
+exports.Trashcan = Trashcan;
+exports.VariableMap = VariableMap;
+exports.VariableModel = VariableModel;
+exports.Variables = Variables;
+exports.VariablesDynamic = VariablesDynamic;
+exports.VerticalFlyout = VerticalFlyout;
+exports.Warning = Warning;
+exports.WidgetDiv = WidgetDiv;
+exports.Workspace = Workspace;
+exports.WorkspaceAudio = WorkspaceAudio;
+exports.WorkspaceComment = WorkspaceComment;
+exports.WorkspaceCommentSvg = WorkspaceCommentSvg;
+exports.WorkspaceDragSurfaceSvg = WorkspaceDragSurfaceSvg;
+exports.WorkspaceDragger = WorkspaceDragger;
+exports.WorkspaceSvg = WorkspaceSvg;
+exports.Xml = Xml;
+exports.ZoomControls = ZoomControls;
+exports.blockAnimations = blockAnimations;
+exports.browserEvents = browserEvents;
+exports.bumpObjects = bumpObjects;
+exports.clipboard = clipboard;
+exports.common = common;
+exports.connectionTypes = connectionTypes;
+exports.constants = constants;
+exports.dialog = dialog;
+exports.inject = inject;
+exports.inputTypes = inputTypes;
+exports.internalConstants = internalConstants;
```

Comment:
Don't export this one--these are all aliased in this file to their old names, but I don't want to make the new (internal) names visible.

Reply:
Done